modules/server/src/main/scala/navigate/server/NavigateEngine.scala

```diff
@@ -108,6 +109,7 @@ trait NavigateEngine[F[_]] {
   def m1LoadAoFigure: F[Unit]
   def m1LoadNonAoFigure: F[Unit]
   def lightpathConfig(from:                          LightSource, to:        LightSinkName): F[Unit]
+  def acquisitionAdj(offset:                         Offset, iaa:            Option[Angle], ipa: Option[Angle]): F[Unit]
```

Comment:
is this method to be called when the user confirms?

Reply:
Yes!